Sources/TABResourceLoader/Services/NetworkDataResourceService.swift

```diff
@@ -38,17 +38,22 @@ open class NetworkDataResourceService<NetworkDataResource: NetworkResourceType &
   }
   
   /**
-   Designated initilizer for NetworkDataResourceService
+   Designated initializer for NetworkDataResourceService
    
    - returns: An new instance
    */
   public required init() {
```

Comment:
yeah change that to pass in the shared session, if a user wants to provide their own service with a different session it is still enabled